_posts/blog/2019-08-01-lab-meeting.md

```diff
@@ -0,0 +1,58 @@
+# Whitaker Lab Meeting | 1 August 2019
+
+This week’s lab meet was petite but by no means picayune! 
+We discussed ways to learn physics, Neurohackademy, and how to build trust and confidence in a group. 
+Thanks Kirstie, Isla, and Sarah for your tips, suggestions, and a super interesting conversation!
+
+### Celebrations and cool things to share:
+
+Kirstie: is finding Neurohackademy great so far – with lots of awesome talks! Here are a select few :point_down:
+  * Codeclimate for readable code: https://codeclimate.com
+  * Intro to git: http://rogerdudler.github.io/git-guide
+  * The most fantastic git resource: https://ohshitgit.com
+  * A bunch of thing you can do with Jupyer: https://blog.jupyter.org/i-python-you-r-we-julia-baf064ca1fb6
+  * A great project template for a research project: https://drivendata.github.io/cookiecutter-data-science
+
+Georgia (and Kirstie) got ethics approval for the Citizen Science Project – the day has finally arrived! YAY! 
+The GitHub repository and survey are now live, and we’ve sent an email to the project’s mailing list to launch it. :rocket:
+
+### Questions we’re thinking about: 
+
+Isla: is in the midst of writing her dissertation, and is wondering how to pinpoint the right level of detail needed.
+Because she’s used to sharing knowledge and making things clear :green_heart:, it can be heard to work out the right level     of explanation for other experts. 
```

Comment:
I'm not sure if the `:green_heart:` syntax renders in jekyll. I think you have to copy the emoji. I do that from https://emojipedia.org/green-heart/

Also fixe weird whitespace (which probably comes from writing in word and then copying and pasting.)